front/components/agent_builder/observability/shared/EmptyPlaceholder.tsx

```diff
@@ -0,0 +1,32 @@
+import { Icon } from "@dust-tt/sparkle";
+import type { ComponentType } from "react";
+
+interface EmptyPlaceholderProps {
+  icon: ComponentType;
+  title: string;
+  description: string;
+}
+
+export function EmptyPlaceholder({
```

Comment:
I think we should use it for testing and feedback too but is it on your radder?

| testing | insights | feedback |
| -------- | ------- | ------- |
|<img width="1248" height="1692" alt="Image" src="https://github.com/user-attachments/assets/b5995c6a-6410-4a0c-9a00-7bed593e4574" />|<img width="1242" height="1698" alt="Image" src="https://github.com/user-attachments/assets/ad5ff393-004b-4c1e-9cf5-8a1037c5d7b3" />|<img width="1248" height="1692" alt="Image" src="https://github.com/user-attachments/assets/7286fd77-0d0f-4aa0-9057-72582fce0a38" />|